lib/fastlane/actions/carthage.rb

```diff
@@ -26,16 +25,21 @@ def self.description
 
       def self.available_options
         [
+          FastlaneCore::ConfigItem.new(key: :command,
+                                       env_name: "FL_CARTHAGE_COMMAND",
```

Comment:
thats a good point. thanks for taking the time for the conversation! I am definitely happy to merge like this! do you mind just squashing up the commits? :100: